main.go

```diff
@@ -2,23 +2,45 @@ package main
 
 import (
 	"context"
+	"flag"
 	"github.com/chzyer/readline"
 	"github.com/disgoorg/disgo"
 	"github.com/disgoorg/disgo/bot"
+	"github.com/disgoorg/disgo/cache"
 	"github.com/disgoorg/disgo/discord"
 	"github.com/disgoorg/disgo/events"
+	"github.com/disgoorg/disgo/gateway"
 	"github.com/disgoorg/disgo/handler"
 	"github.com/disgoorg/disgo/handler/middleware"
+	"github.com/nitkgsm-dev/labot/pkg/logging"
 	"log/slog"
 	"os"
+	"time"
 )
 
+var (
+	jsonLog    bool
+	token      string
+	debug      bool
+	dateFormat string
+)
+
+func init() {
+	flag.BoolVar(&jsonLog, "json-log", false, "enable json log format")
+	flag.StringVar(&token, "token", "", "discord bot token")
+	flag.StringVar(&dateFormat, "date-format", time.TimeOnly, "date format")
+	flag.BoolVar(&debug, "debug", false, "enable debug mode")
+	flag.Parse()
+}
+
```

Comment:
```suggestion
var (
	jsonLog    = flag.Bool("json-log", false, "enable json log format")
	token      = flag.String("token", "", "discord bot token")
	debug      = flag.Bool("debug", false, "enable debug mode")
	dateFormat = flag.String("date-format", time.TimeOnly, "date format")
)

func init() {
	flag.Parse()
}
```

バラバラに初期化するよりもこっちのほうが、いいと思う。